### DISCUSSION TITLE
Discounts

### DISCUSSION THREAD

**Discussion by Isaev2992** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#900] [Show and tell]
Hello. I saw that sometimes you distribute promotional codes for the purchase of the AlDente program.
When is the next discount?
Thank you!

**Comment by MatthiasKerbl** [comment]
Hi @Isaev2992,

Unfortunately, we do not have any plans for upcoming discounts on AlDente Pro at the moment.
Maybe we will have some kind of offer during Black Friday again but no specific plans yet.

Cheers,
Matthias

**Reply by Isaev2992** [reply]
Thanks for the answer!

**Reply by senhalil** [reply]
Where would you announce any possible discount?

**Reply by MatthiasKerbl** [reply]
On our website.

**Comment by Roywuwu** [comment]
Hi @MatthiasKerbl 

May I know is there any promotion in Black Friday this year?
Thx
